iphone/TitaniumKit/TitaniumKit/Sources/API/UIImage+Compare.m

```diff
@@ -0,0 +1,111 @@
+/**
+* Appcelerator Titanium Mobile
+* Copyright (c) 2020-Present by Appcelerator, Inc. All Rights Reserved.
+* Licensed under the terms of the Apache Public License
+* Please see the LICENSE included with this distribution for details.
+*/
+
+#import "UIImage+Compare.h"
+
+typedef union {
+  uint32_t raw;
+  unsigned char bytes[4];
+  struct {
+    char red;
+    char green;
+    char blue;
+    char alpha;
+  } __attribute__((packed)) pixels;
+} ImagePixel;
+
+@implementation UIImage (Compare)
+
+- (BOOL)compareWithImage:(UIImage *)image tolerance:(CGFloat)tolerance
```

Comment:
I'd prefer we didn't add this as API. It's a fine workaround because of the perf issue for comparing images in #11810  - but I'd prefer we consider including that fix for 9.2.0 and just use it rather than implement an undocumented one-off API for comparing two images natively. cc @mukherjee2 

But who knows, maybe developers want this?